tests/integration/Legacy/Metadata/ParserTest.php

```diff
@@ -43,9 +44,10 @@ public function setup()
         $this->logger = m::mock(LoggerInterface::class);
 
         $rootDir = __DIR__.'/../../../../app/Resources/';
+        $attributeRepository = new AttributeRepository(__DIR__ . '/../../../../app/config/attributes.json');
```

Comment:
Is using the real config file the way to go here? I'd propose you copy paste it to a `fixtures` folder in the folder where this test resides.

This pattern also emerges in: 
- EntityMergeServiceTest.php
- ArpGeneratorTest.php